tools/webpack

```diff
@@ -55,7 +57,22 @@ def build_for_dev_server(host, port, minify, disable_host_check):
         webpack_args.append('--optimize-minimize')
     if disable_host_check:
         webpack_args.append('--disable-host-check')
-    os.execvp(webpack_args[0], webpack_args)
+    webpack_process = subprocess.Popen(webpack_args)
+
+    class WebpackConfigFileChangeHandler(pyinotify.ProcessEvent):
+        def process_default(self, event):
+            # type: (Any) -> None
+            nonlocal webpack_process
+            print('Restarting webpack-dev-sever due to config changes...')
+            webpack_process.kill()
```

Comment:
`webpack_process.terminate()` would be friendlier. And `webpack_process.wait()` is needed to avoid accumulating zombie processes.

Reply:
Ah, I tried using `webpack_process.terminate()` first but it did work because I was not using `webpack_process.wait()` and so caused the new webpack process to start before the old one was terminated.